CoreFoundation/NumberDate.subproj/CFBigNumber.c

```diff
@@ -9,6 +9,7 @@
 	Original author: Zhi Feng Huang
 */
 
+#include <CoreFoundation/TargetConditionals.h>
```

Comment:
We cant do this since it only exists on non-darwin platforms. The Darwin variant is just <TargetConditionals.h> additionally the proper TargetConditionals header should be included by CFBase.h so any additions should not be needed.

This applies to all other instances of the addition of this header.

Reply:
even though TARGET_RT_64_BIT is ok, the CoreFoundation/TargetConditionals thing still holds. These header includes would be a breaking change for cycling back to the Darwin implementation.

Reply:
Okay, I'll adjust the sites to use `CFBase.h` instead.  The locations that I included this header were not already including `CFBase.h`.